doc/modules/feature_extraction.rst

```diff
@@ -398,18 +398,34 @@ suitable for usage by a classifier it is very common to use the tf–idf
 transform.
 
 Tf means **term-frequency** while tf–idf means term-frequency times
-**inverse document-frequency**. This was originally a term weighting
-scheme developed for information retrieval (as a ranking function
-for search engines results), that has also found good use in document
-classification and clustering.
+**inverse document-frequency**: 
+:math:`\text{tf-idf(t,d)}=\text{tf(t,d)} \times \text{idf(t)}`. 
+
+This was originally a term weighting scheme developed for information retrieval 
+(as a ranking function for search engines results) that has also found good 
+use in document classification and clustering.
+
+Please note that the tf-idfs computed in scikit-learn's 
+:class:`TfidfTransformer` and :class:`TfidfVectorizer` differ slightly from the
+standard textbook notation, which is typically defines the idf as as
```

Comment:
This was a bit ambiguous to me upon first reading it, i think it'd be more clear to say "differ slightly from the standard textbook notation that defines the idf as:", since i wasn't sure initially if you were referring to the textbook notation or the scikit-learn notation.

If you think it's ok the way it is now, can you at least remove the "is" and the duplicated "as"?